blockchain_connector/common/connector_common/work_order_delegate.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import random
+import asyncio
+import logging
+from error_code.error_status import WorkOrderStatus
+from avalon_sdk.connector.blockchains.common.contract_response \
+    import ContractResponse
+
+logging.basicConfig(
+    format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO)
+
+
+class WorkOrderDelegate():
+
+    """
+    Helper class to sync work orders between avalon and blockchain
+    """
+
+    def __init__(self, jrpc_work_order, work_order_instance):
+        """
+        Initialize the connector with instances of jrpc worker
+        implementation and blokchain worker implementation objects.
+        @params
```

Comment:
Remove @params line and add @param in front of lines and 39 below.

Reply:
Done